addons/web/views/report_templates.xml

```diff
@@ -266,50 +266,47 @@
 
     <template id="minimal_layout">
         &lt;!DOCTYPE html&gt;
-        <t t-raw="'&lt;base href=%s&gt;' % base_url"/>
         <html style="height: 0;">
             <head>
+                <base t-att-href="base_url"/>
                 <t t-call-assets="web.report_assets_pdf" t-js="false"/>
                 <t t-call-assets="web.report_assets_common" t-js="false"/>
                 <t t-call-assets="web.report_assets_pdf" t-css="false"/>
                 <meta charset="utf-8"/>
-                <t t-set="subst_needed" t-value="subst is True"/>
-                <t t-if="subst_needed">
-                    <script>
-                        function subst() {
-                            var vars = {};
-                            var x = document.location.search.substring(1).split('&amp;');
-                            for (var i in x) {
-                                var z = x[i].split('=', 2);
-                                vars[z[0]] = unescape(z[1]);
-                            }
-                            var x = ['sitepage', 'sitepages', 'section', 'subsection', 'subsubsection'];
-                            var z = {'sitepage': 'page', 'sitepages': 'topage'};
-                            for (var i in x) {
-                                var y = document.getElementsByClassName(z[x[i]] || x[i])
-                                for (var j=0; j&lt;y.length; ++j)
-                                    y[j].textContent = vars[x[i]];
-                            }
+                <script>
+                    function subst() {
```

Comment:
why not keep t-if='subst' to avoid to have this code when not called ?

Reply:
Sure, why not.